CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog
 0.12.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Add transport header support for thrift call.
+- Fix zipkin trace.
```

Comment:
Can you be more descriptive? What about zipkin trace was broken? Can we not fix this in a patch release without introducing any new functionality?